yarn.lock

```diff
@@ -2,94 +2,152 @@
 # yarn lockfile v1
```

Comment:
Why did everything get upgraded? Not a bad thing, just surprised since I didn't see anything new to `package.json`.

Reply:
It didn't do any major updates. There are some that need to be done, but I need to see what breaking changes could have been introduced. There were some minor and patch updates.